friends/src/App.js

```diff
@@ -1,73 +1,42 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import axios from 'axios'
 import Friend from './Friend/Friend';
 import NewFriend from './NewFriend/NewFriend';
 import './App.scss';
 
-class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      friends: [],
-      postRes: '',
-      friend: {
-        name: '',
-        age: 0,
-        email: ''
-      }
-    }
-  }
+const App = () => {
+  const [friends, setFriends] = useState([]);
 
-  componentDidMount() {
-    axios
-      .get('http://localhost:5000/friends')
-      .then(res => this.setState({
-        friends: res.data
-      }))
-      .catch(err => console.log(err))
-  }
+  // CDM
```

Comment:
As mentioned in a previous review, use `[]` as a second argument to `useEffect` for CMD life cycle